src/components/BookstoreContainer.js

```diff
@@ -7,7 +7,7 @@ import Categories from './Categories';
 
 function BookstoreContainer() {
   return (
-    <div className="bookstore-container">
+    <div className="bookstore-container" style={{ backgroundColor: '#fafafa' }}>
```

Comment:
- [ ] It would be better if our UI looks like the figma design. Let us kindly compare our design with the original one and let us humbly solve all the differences including the placeholders

### Original
![Screenshot from 2022-09-02 12-23-45](https://user-images.githubusercontent.com/46388113/188146318-4b4d038f-cf38-4111-b245-bd245274adb4.png)


### Our Design

![Screenshot from 2022-09-02 12-24-09](https://user-images.githubusercontent.com/46388113/188146193-dda8639f-70e6-4417-b4cf-b38d930ee3ea.png)